telegram/messageentity.py

```diff
@@ -90,36 +90,36 @@ def de_json(cls, data: Optional[JSONDict], bot: 'Bot') -> Optional['MessageEntit
 
         return cls(**data)
 
-    MENTION: str = 'mention'
-    """:obj:`str`: 'mention'"""
-    HASHTAG: str = 'hashtag'
-    """:obj:`str`: 'hashtag'"""
-    CASHTAG: str = 'cashtag'
-    """:obj:`str`: 'cashtag'"""
-    PHONE_NUMBER: str = 'phone_number'
-    """:obj:`str`: 'phone_number'"""
-    BOT_COMMAND: str = 'bot_command'
-    """:obj:`str`: 'bot_command'"""
-    URL: str = 'url'
-    """:obj:`str`: 'url'"""
-    EMAIL: str = 'email'
-    """:obj:`str`: 'email'"""
-    BOLD: str = 'bold'
-    """:obj:`str`: 'bold'"""
-    ITALIC: str = 'italic'
-    """:obj:`str`: 'italic'"""
-    CODE: str = 'code'
-    """:obj:`str`: 'code'"""
-    PRE: str = 'pre'
-    """:obj:`str`: 'pre'"""
-    TEXT_LINK: str = 'text_link'
-    """:obj:`str`: 'text_link'"""
-    TEXT_MENTION: str = 'text_mention'
-    """:obj:`str`: 'text_mention'"""
-    UNDERLINE: str = 'underline'
-    """:obj:`str`: 'underline'"""
-    STRIKETHROUGH: str = 'strikethrough'
-    """:obj:`str`: 'strikethrough'"""
+    MENTION: str = constants.MESSAGEENTITY_MENTION
+    """:const:`telegram.constants.MESSAGEENTITY_MENTION`"""
+    HASHTAG: str = constants.MESSAGEENTITY_HASHTAG
+    """:const:`telegram.constants.MESSAGEENTITY_HASHTAG`"""
+    CASHTAG: str = constants.MESSAGEENTITY_CASHTAG
+    """:const:`telegram.constants.MESSAGEENTITY_CASHTAG`"""
+    PHONE_NUMBER: str = constants.MESSAGEENTITY_PHONE_NUMBER
+    """:const:`telegram.constants.MESSAGEENTITY_PHONE_NUMBER`"""
+    BOT_COMMAND: str = constants.MESSAGEENTITY_BOT_COMMAND
+    """:const:`telegram.constants.MESSAGEENTITY_BOT_COMMAND`"""
+    URL: str = constants.MESSAGEENTITY_URL
+    """:const:`telegram.constants.MESSAGEENTITY_URL`"""
+    EMAIL: str = constants.MESSAGEENTITY_EMAIL
+    """:const:`telegram.constants.MESSAGEENTITY_EMAIL`"""
+    BOLD: str = constants.MESSAGEENTITY_BOLD
+    """:const:`telegram.constants.MESSAGEENTITY_BOLD`"""
+    ITALIC: str = constants.MESSAGEENTITY_ITALIC
+    """:const:`telegram.constants.MESSAGEENTITY_ITALIC`"""
+    CODE: str = constants.MESSAGEENTITY_CODE
+    """:const:`telegram.constants.MESSAGEENTITY_CODE`"""
+    PRE: str = constants.MESSAGEENTITY_PRE
+    """:const:`telegram.constants.MESSAGEENTITY_PRE`"""
+    TEXT_LINK: str = constants.MESSAGEENTITY_TEXT_LINK
+    """:const:`telegram.constants.MESSAGEENTITY_TEXT_LINK`"""
+    TEXT_MENTION: str = constants.MESSAGEENTITY_TEXT_MENTION
+    """:const:`telegram.constants.MESSAGEENTITY_TEXT_MENTION`"""
+    UNDERLINE: str = constants.MESSAGEENTITY_UNDERLINE
+    """:const:`telegram.constants.MESSAGEENTITY_UNDERLINE`"""
+    STRIKETHROUGH: str = constants.MESSAGEENTITY_STRIKETHROUGH
+    """:const:`telegram.constants.MESSAGEENTITY_STRIKETHROUGH`"""
     ALL_TYPES: List[str] = [
```

Comment:
Can you move this, too? Overlooked in the last review

Reply:
Remove it? Or what? I don't understand.

Reply:
I mean move `ALL_TYPES` to `tg.constants`, too

Reply:
And ALL_EMOJI in Dice too, right?

Reply:
uh, yes :D good catch!